hm_1/tasks/task_01_05.py

```diff
@@ -0,0 +1,17 @@
+"""
+Given a list of integers numbers "nums".
+You need to find a sub-array with length less equal to "k", with maximal sum.
+The written function should return the sum of this sub-array.
+Examples:
+    nums = [1, 3, -1, -3, 5, 3, 6, 7], k = 3
+    result = 16
+"""
+from typing import List
+
+
+def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
+    sums = []
+    for j in range(k, -1, -1):
```

Comment:
You can traverse in any direction here.  From 1 to k is more readable in that case.